### PR TITLE
SUSE Manager: Universe is not required for installing spacecmd on Ubuntu

### PR DESCRIPTION
**Pull request by juliogonzalez**
# Description

We already provided the require binaries for Ubuntu 18.04, but the channels definitions were wrong. That will be fixed for 4.2.2.

For Ubuntu 20.04 will start providing the packages as well.

So no more Universe requirement for SUSE Manager.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.2
- [x] Manager-4.1
- [ ] Manager-4.0

# Links

https://github.com/SUSE/spacewalk/issues/15199

### CODE REVIEW

File: .changelog [hunk 1]
```diff
@@ -4,6 +4,11 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- SUSE Manager: Universe is not required for installing spacecmd
+  on Ubuntu (bsc1187337)
+
+# Last packaged: ?
```
Comment (juliogonzalez) [inline .changelog:10]: I am not sure what do do here. I think Joseph should have cleaned this up when he packaged last time. Anyway, please fix it on a separate PR :-)

Comment (keichwa) [reply .changelog:10]: :+1: if nobody knows better, we can write `Last packaged: before 2021-08-18`.

**Review by keichwa — APPROVED** [review]
thanks for the cleanup.  Looks save!

File: .changelog [hunk 2]
```diff
@@ -4,8 +4,13 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- SUSE Manager: Universe is not required for installing spacecmd
+  on Ubuntu (bsc1187337)
 - In the Prometheus chapter of the Administration Guide advise to store
   data locally (bsc#1188855)..
```
Comment (juliogonzalez) [inline .changelog:10]: @keichwa I guess that part of the changelog is not right, as it contains two dots ^, make sure you fix it so Joseph doesn't need to take care of it on packaging day :-)

**Comment by juliogonzalez** [comment]
@keichwa FYI: Seems what I am removing here was never backported to 4.2 or 4.1, so need to backport this PR either.

**Pull request merged**